DigitalLearningSolutions.Web/Views/MyAccount/Index.cshtml

```diff
@@ -0,0 +1,67 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.MyAccount
+@using Microsoft.Extensions.Configuration
+@model MyAccountViewModel
+
+@inject IConfiguration Configuration
+
+@{
+  ViewData["Title"] = "My Account";
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <h1 id="page-heading">My account</h1>
+
+    <div class="nhsuk-card">
+      <div class="nhsuk-card__content">
+        <dl class="nhsuk-summary-list">
+          <div class="nhsuk-summary-list__row">
+            <dt class="nhsuk-summary-list__key">
+              Centre:
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+              @Model.Centre
+            </dd>
+          </div>
+
+          <div class="nhsuk-summary-list__row">
+            <dt class="nhsuk-summary-list__key">
+              User:
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+              @Model.User

```

Comment:
Can we also check if the Model.User exists, before presenting this. In the live system, I don't see this field when I log in with my delegate account (I do see it with my admin account), but I'm not sure why it's not set on the claims. I will investigate, but I think the best thing to do is to handle the case where it's null :)